### DISCUSSION TITLE
Chrome extension?

### DISCUSSION THREAD

**Discussion by HenriqueDerosa** [TanStack/query#3445] [Ideas]
Is there any initiative to make a browser extension?

Since the devtool is being render inside the application itself, sometimes I felt it was harder to manage its placement on the screen in small monitors. 

An extension to the Chrome DevTool would be great, similar to what we have with Redux Devtools. :)

**Comment by TkDodo** [comment]
There are no plans for now. Doing a browser extension is a ton of work :)

**Reply by HenriqueDerosa** [reply]
Indeed 😅 

Another reason to make it is that the current devtools breaks with the application, when there's a problem. Which becomes harder to debug that.

Not sure if there's a workaround for this situation.

**Reply by TkDodo** [reply]
ErrorBoundaries?

**Comment by feledori** [comment]
I just started a RN project and was looking for the query devtools and saw that there are community repos doing plugins for tools.
As of [RFC-0641](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0641-decoupling-flipper-from-react-native-core.md) its no long recommended to use flipper and use chrome dev tools. I know its a lot of work but having a chrome devtools plugin would make the devtools platform independent.

**Comment by AurelianSpodarec** [comment]
;(

**Comment by dborysov** [comment]
There is now https://chromewebstore.google.com/detail/tanstack-query-devtools/annajfchloimdhceglpgglpeepfghfai. I'd be thrilled to hear some feedback

**Reply by mavalos1** [reply]
> There is now https://chromewebstore.google.com/detail/tanstack-query-devtools/annajfchloimdhceglpgglpeepfghfai. I'd be thrilled to hear some feedback

Just used this today and have to say it's awesome that we'd have an extension now 🚀. Not perfect with Next's fast refresh but certainly useful 🦾

Thank you @dborysov

**Reply by dborysov** [reply]
Thanks @mavalos1!

TBH, I didn't test it with the Nextjs project. Could you share the steps I could use to reproduce the problem?

I don't think this thread is a good place to discuss the issue in a "community supported" product. Could you please leave a review with the steps in the extension page itself? Or send me an email? I'll do my best to fix it in the following days

Thanks in advance! 🙏🏻

**Reply by Tarunvamsi** [reply]
> There is now https://chromewebstore.google.com/detail/tanstack-query-devtools/annajfchloimdhceglpgglpeepfghfai. I'd be thrilled to hear some feedback

Could you please share the steps. how to enable this extension. i tried installing, but was unable to use with my webpage where i used tanstackquery

**Reply by reneleonhardt** [reply]
Unfortunately discussions are disabled, can you open a reproducable installation issue?
https://github.com/DeeCode-inc/tanstack-query-chrome-devtools/issues

**Reply by Tarunvamsi** [reply]
I was able to install it, and it's working fine. Thank you for the extension it has made debugging much easier.

**Reply by dborysov** [reply]
> Unfortunately discussions are disabled, can you open a reproducable installation issue? https://github.com/DeeCode-inc/tanstack-query-chrome-devtools/issues

I enabled the [discussions](https://github.com/DeeCode-inc/tanstack-query-chrome-devtools/discussions)
